### PR TITLE
fixed: devtools styles disabled donot sync

### PR DESCRIPTION
**Pull request by tinys**
fixed devtools css panel styles uncheckd dont sync

### CODE REVIEW

**Review by yifei8 — APPROVED** [review]

File: webf/lib/src/devtools/modules/css.dart [hunk 1]
```diff
@@ -75,6 +74,30 @@ class InspectCSSModule extends UIInspectorModule {
     }
   }
 
+  Map<String, dynamic> parseAnnotatedStyle(String text) {
+    Map<String, String> styles = {};
```
Comment (andycall) [inline webf/lib/src/devtools/modules/css.dart:78]: Why not use CSSParser instead of write your own?

Comment (tinys) [reply webf/lib/src/devtools/modules/css.dart:78]: function(parseAnnotatedStyle) have two purpose
1、parse should disabled property
2、get a clean style text ,without annotation
CSSParser used to be parse style property from a text，I don't know which property should be removed；

**Comment by andycall** [comment]
https://user-images.githubusercontent.com/4409743/198312051-da539d6f-3689-44c9-bb4f-ad5e17747045.mov

This looks wired.

File: webf/lib/src/devtools/modules/css.dart (hunk 1 above)
Comment (tinys) [reply webf/lib/src/devtools/modules/css.dart:79]: fixed bugs;

**Comment by andycall** [comment]
Still not working..

**Pull request closed without merging**